common/src/app/common/pages/changes_builder.cljc

```diff
@@ -52,39 +63,58 @@
 
 (defn- assert-objects
   [changes]
-  (assert (contains? (meta changes) ::objects) "Call (with-objects) before using this function"))
+  (assert (contains? (meta changes) ::file-data) "Call (with-objects) before using this function"))
+
+(defn- apply-changes-local
+  [changes]
+  (if-let [file-data (::file-data (meta changes))]
+    (let [index         (::applied-changes (meta changes))
```

Comment:
the `::applied-changes` looks like a wring name, because it does not contains changes...

Reply:
changed to applied-changes-count